### DISCUSSION TITLE
Some sentences that consist of '&' are being cut off when performing over the model 'en_core_web_trf'

### DISCUSSION THREAD

**Discussion by qingyun1988** [explosion/spaCy#8485] [Language Support]
For example：
`So important is agriculture RD&E that the Coalition Government went to the 2013 Election with a policy to increase RD&E by $100 million in the agriculture sector.`

![image](https://user-images.githubusercontent.com/38597436/123063746-4248e100-d440-11eb-99e7-d4bf13f4848f.png)

**Comment by adrianeboyd** [comment]
Hi, how are you passing in this text? My first guess would be that's it's an interaction with your terminal shell and the `&` character, but it's hard to know without more details. The pretrained pipelines should never truncate texts.

**Reply by qingyun1988** [reply]
Thanks for your reply! 

This is my fault. I have built a local website for parsing dependency, but the URL params aren't encoded by encodeURIComponent in javascript. 

I'm very sorry!
